swap_meet/vendor.py

```diff
@@ -1,2 +1,96 @@
+from queue import PriorityQueue
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory  = None):
+        if not inventory:
+            inventory = []
```

Comment:
No real need to reassign inventory to an empty array first. If it's `None`, you could just say `self.inventory = []`. Or you could capture this in a ternary expression:
```python
self.inventory = inventory if inventory is not None else []

```